job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/Job.groovy

```diff
@@ -617,6 +617,39 @@ abstract class Job extends Item {
         }
     }
 
+    /**
+     * Enables job rebuild functionality
+     *
+     * @since 1.39
+     */
+    @RequiresPlugin(id = 'rebuild', minimumVersion = '1.25')
+    void enableJobRebuild(boolean automaticallyUseParmetersFromPreviousRun = false) {
```

Comment:
A `PropertiesContext` has been added a while ago. All new DSL methods creating elements below `<properties>` should be added there.